views/layout.etlua

```diff
@@ -14,7 +14,17 @@
   </head>
   <body>
     <header id="header">
-      <img style='height: 100%; width: 10%; object-fit: contain; align=left' src="static/images/mudlet_192px.png">
+      <div class="container-fluid">
+          <div class="row">
+            <div class="col-2">
+              <img style='height: 100%; width: 30%; object-fit: contain; align=left' src="static/images/mudlet_192px.png" alt="Mudlet logo">
```

Comment:
The height and width here is pretty hacky, but I gave it a go but couldn't coerce it to be any better.